cumulus_library/cli.py

```diff
@@ -179,6 +195,15 @@ def run_cli(args: Dict):
             builder.verbose = True
 
         study_dict = get_study_dict(args["study_dir"])
+        if args["target"]:
+            for target in args["target"]:
+                if target not in study_dict:
```

Comment:
This feels awkward, but I'm not sure I love some alternatives I thought of (do a `set` difference, maybe define `target = args["target"] or []` so you don't need the `if` AND `for`....)

Reply:
i could move it into a function to hide the awardness from this primary entrypoint function?